### ISSUE TITLE
A11y_Azure Extensions for VS Code_Functions (Disable Function)_Keyboard: on navigating to the configure button the notification and pressing enter an item opens and collapse it we press escape on doing so the keyboard focus does not return to the configure button and on tab is outside the notification

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode-azurefunctions#3637]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHub Tags:**#A11yTCS;#Azure Tools for VS Code;#A11y_Azure Extensions for VS Code; #DesktopApp;#Win11;#MAS2.4.3;#Benchmark;#FTP;#A11yMAS;#A11ySev2;#AINotInScope;#Keyboard;#LinkedBug;#Linked:Bug184711;

## Environment Details:
Application Name: Azure Extensions for VS Code
Version: 1.77.0-insider (user setup)
Commit: fa13bf16fd237afef07f4df0648da026f43e3963
Date: 2023-03-20T05:25:30.711Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT x64 10.0.25300
Sandboxed: Yes

widows version:
Edition	Windows 11 Enterprise Insider Preview
Version	22H2
Installed on	‎21-‎02-‎2023
OS build	25300.1000
Experience	Windows Feature Experience Pack 1000.25300.1000.0

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.
2. Open Visual studio code
3. Click on extension button and search for "Azure Functions"
4. Click on install and move to the "Azure" button
5. Click on"Local Project" from the treeview and expand it.
6. Click on the Functions menu and expand it further to see Triggers.
7. Right click on trigger to see the menu item "Disable Function"
8. Navigate to configure notification button and expand and then collapse with the help of esc key.

## Actual:

on navigating to the configure button the notification and pressing enter an item opens and collapse it we press escape on doing so the keyboard focus does not return to the configure button and on tab is outside the notification

Same issue is also observed with throughout the application.

## Expected:

Focus should be remain on the configure button on collapsing the list

## User Impact:

User will not be able to return back to the button from where they performed the action

## Attachment:
[2225.zip](https://github.com/microsoft/vscode-azurefunctions/files/11077102/2225.zip)

**Comment by fsteffi** [comment]
logged in ref to bug https://github.com/Microsoft/vscode-azurefunctions/issues/2225

**Comment by MicroFish91** [comment]
@fsteffi, Eric previously filed an issue upstream with VS Code [here](https://github.com/microsoft/vscode/issues/102983) that would address this.

You may need to message the VS Code team and have them re-open (https://github.com/microsoft/vscode/issues/102983) as this issue is not something the extension can solve.  We previously closed the issue posted against our extension after some time because the issue we were tracking against was closed.

**Comment by MicroFish91** [comment]
Hi @fsteffi I think the example you used isn't a 1:1 repro because `Disable Function` uses an expandable `showInformationMessage` window.

In the example below, I'm running my `a11y-kit` extension which you can download on the VS Code marketplace.  It's an accessibility test extension I built that uses barebones VS Code UI calls so it helps us to isolate issues without all the extra logic we use in our azure extensions.  You can install and run F1 > a11y-kit: Select UI feature for accessibility testing.

You can see below that when an `expandable` window is in the `collapsed` state, the issue will reproduce.  When that same window is `expanded`, it will not reproduce.

Because of this, I think the issue would still need to be opened upstream to resolve this.

![#3637](https://github.com/microsoft/vscode-azurefunctions/assets/40250218/a816422c-2b69-4b41-a761-e35552bcd610)

**Comment by fsteffi** [comment]
external bug https://github.com/microsoft/vscode/issues/184711

**Comment by MicroFish91** [comment]
Hi @fsteffi,

It looks like the upstream issue was closed, are we good to close this issue out as well?

**Comment by fsteffi** [comment]
#closed;
[Video 6-14 at 16.15.webm](https://github.com/microsoft/vscode/assets/24580570/03134505-bd5c-430b-b039-b9427967651c)

Version: 1.80.0-insider (user setup)
Commit: https://github.com/microsoft/vscode/commit/f3aa9a201b2455e6556797e8ffb7145d77adb792
Date: 2023-06-14T09:02:23.566Z
Electron: 22.5.7
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.25375
